llvm/lib/Transforms/Coroutines/CoroEarly.cpp

```diff
@@ -152,7 +152,8 @@ void Lowerer::lowerCoroNoop(IntrinsicInst *II) {
   }
 
   Builder.SetInsertPoint(II);
-  auto *NoopCoroVoidPtr = Builder.CreateBitCast(NoopCoro, Int8Ptr);
+  auto *NoopCoroVoidPtr =
+      Builder.CreateAddrSpaceCast(NoopCoro, PointerType::getUnqual(C));
```

Comment:
You can't insert any addrspacecast you want. Really these intrinsics should be fixed to have an address space mangled pointer. We should not have any intrinsics defined with llvm_ptr_ty